content/en/logs/log_collection/csharp.md

```diff
@@ -260,9 +260,13 @@ If, despite the benefits of logging in JSON, you wish to log in raw string forma
 {{% /tab %}}
 {{< /tabs >}}
 
-**Connect Logs and Traces**
+## Connect your service across Logs and APM
 
-If APM is enabled for this application, the correlation between application logs and traces can be improved by [following APM .NET logging instructions][2] to add trace and span IDs in your logs.
+If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
+See [APM .NET logging instructions][2] to learn more about adding trace IDs, span IDs, and even the
+`env`, `service`, and `version` to your logs.
```

Comment:
What does `APM data` means? Are we talking about `traces`?

I would recommend to merge the two sentences (actually remove the first one as it doesn't bring much compared to the second).

```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM .NET instructions][2]
```